java.net.BindException: Can't assign requested address (Bind failed)

I get the following error when attempting to start gretty:

```
Caused by: java.net.BindException: Can't assign requested address (Bind failed)
        at org.akhikhl.gretty.ServiceProtocol.createReader(ServiceProtocol.groovy:20)
```

Looking at line 20 of ServiceProtocol.groovy in gretty-common, it's using InetAddress.localHost, rather InetAddress.loopbackAddress.  On a mac at least, InetAddress.localHost apparently uses the hostname to look up the IP address to bind to.  Unfortunately, I'm on a VPN which is handing out a hostname that currently doesn't resolve to the assigned IP address, so the IP gretty is attempting to bind to isn't bound to any interface on my system. 

Additionally, I don't really want the gretty ports running where I'd have to firewall them.  It seems to me that using InetAddress.loopbackAddress would be more secure and less error prone.

--- a/libs/gretty-common/src/main/groovy/org/akhikhl/gretty/ServiceProtocol.groovy
+++ b/libs/gretty-common/src/main/groovy/org/akhikhl/gretty/ServiceProtocol.groovy
@@ -17,7 +17,7 @@ import java.util.concurrent.CompletableFuture
 final class ServiceProtocol {
 
   static Reader createReader(int port  = 0) {
-    final ServerSocket serverSocket = new ServerSocket(port, 1, InetAddress.localHost)
+    final ServerSocket serverSocket = new ServerSocket(port, 1, InetAddress.loopbackAddress)
     return new Reader(serverSocket)
   }
 
@@ -73,7 +73,7 @@ final class ServiceProtocol {
 
     def write(final String command) {
       log.debug 'ServiceProtocol.send({}, {})', port, command
-      Socket s = new Socket(InetAddress.localHost, port)
+      Socket s = new Socket(InetAddress.loopbackAddress, port)
       try {
         OutputStream out = s.getOutputStream()
         out.write(("${command}\n<<EOF>>\n").getBytes(StandardCharsets.UTF_8))

